### ISSUE TITLE
Problems with 10.0.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4346]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4346|
|Type|defect|
|Created|2010-03-03 22:00:16Z|
|Branch|https://github.com/twisted/twisted/tree/ten-problems-4346|

  1. The `INSTALL` file (Skipping the things from [#4344](https://github.com/twisted/twisted/issues/4344)):
    1. Zope Interface seems to be up to about version 3.5.  The URL we link to has also been abandoned for some time.
    1. As far as I can tell, we are no longer going to be distributing Windows .exe installers.  Instead, it seems we'll be building MSIs.
  1. The `README` file:
    1. The list of foreign event loops we integrate with is a bit out of date, ''Tk, GTK+, GTK+ 2, Qt, Mac OS X, or wxPython''
      1. We don't support Qt anymore (although hopefully that will change again)
      1. There is no '''supported''' OS X or wxPython integration.  There's some experimental, half-working, half-broken reactors.  We should differentiate these from the '''supported''' event loops.
    1. ''Examples on how to use Twisted APIs are located in doc/examples'' - this looks like the pre-split examples directory; `doc/core/examples` is probably where it should be pointing.  Or it should have multiple pointers, to each subproject's examples.
    1. Likewise, `doc/howto/index.xhtml` is not a valid path.
    1. What's with the ''::'' all over?
    1. I think the preferred hostname for Freenode is `chat.freenode.net`.
  1. `twisted/__init__.py` contains a Python version check intended to prevent use on Python versions.  It still allows Python 2.3, though this is no longer officially supported.

<details><summary>Searchable metadata</summary>

```
trac-id__4346 4346
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__Twisted_10_0 Twisted-10.0
branch__branches_ten_problems_4346 branches/ten-problems-4346
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1267653616000000 1267653616000000
changetime__1269174011000000 1269174011000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_1"></a>|@jml commented|
|-|-|

Do we need to update the Zope thing? I mean, will we still work w/ 3.0.1?

The double colons are ReST hangovers, presumably

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @jml|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

> Do we need to update the Zope thing? I mean, will we still work w/ 3.0.1? 

I have no idea what versions of Zope Interface we support.  There's no way to find out what version is installed on the slaves except to ask the operators.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_4"></a>|@jml commented|
|-|-|

I guess for zope.interfaces, we just leave the version in the docs as they are until we have evidence of them being wrong.

The [PyPI page for zope.interface](http://pypi.python.org/pypi/zope.interface/) seems a good place to link to instead of the zope.org one.

I'm putting this on the 10.0 milestone, since [#4210](https://github.com/twisted/twisted/issues/4210) and [#2764](https://github.com/twisted/twisted/issues/2764) make me think we should consider a 10.0.1 release.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Okay, I managed to extract version information from some of the slaves.  This is now available alongside all the rest of the automatically extracted version information (go to a build page, look at the versions step (usually the second) and follow the "versions" link on it).

Not all the slaves can report their version of zope.interface, but two of them did manage to, and reported 3.3.0 and 3.3.1.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_6"></a>|@jml commented|
|-|-|

See r28716

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess set owner to @jml|
|-|-|

__init__.py needs a copyright bump. Other than that, looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_9"></a>|@jml set status to `closed`|
|-|-|

(In [28759]) Merge ten-problems-4346

 * Author: jml
 * Reviewer: jesstess
 * Fixes [#4346](https://github.com/twisted/twisted/issues/4346)

Fix some small but important defects in release-level documentation

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
